tracer/src/Datadog.Trace/Ci/Agent/CIWriterHttpSender.cs

```diff
@@ -239,13 +245,11 @@ private async Task SendPayloadAsync(MultipartPayload payload)
                 payloadBytes += multipartFormItem.ContentInBytes?.Count ?? 0;
             }
 
-            TelemetryFactory.Metrics.RecordCountCIVisibilityEndpointPayloadRequests(payload.TelemetryEndpoint);
             TelemetryFactory.Metrics.RecordDistributionCIVisibilityEndpointPayloadBytes(payload.TelemetryEndpoint, payloadBytes);
             TelemetryFactory.Metrics.RecordDistributionCIVisibilityEndpointPayloadEventsCount(payload.TelemetryEndpoint, payload.Count);
 
-            var sw = Stopwatch.StartNew();
             Log.Debug<int>("Sending {Count} multipart items...", payload.Count);
-            var statusCode = await SendPayloadAsync(
```

Comment:
If you're not using the return value any more, maybe change the `SendPayloadAsync` too? It's kinda hard to know whether the metrics are right otherwise (as you don't know if the callee is handling the status codes)

Reply:
makes sense, done in [fd2e4a9](https://github.com/DataDog/dd-trace-dotnet/pull/4681/commits/fd2e4a966a81dc3ef9ed3aef883b5fb7aef735ec)